README.md

```diff
@@ -9,7 +9,7 @@ HTML
 CSS
 
 # LIVE DEMO
-https://donard97.github.io/Personal-Portofolio/
+ https://donard97.github.io/Personal-Portofolio/
```

Comment:
- [ ] Your hosted page on GitHub pages is not displaying correctly. From the screenshot below we can see that your mobile version overlaps/ overflows, and therefore does not fit the screen width as intended. We actually want our want our design to look closely as the chosen [Figma template](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988&scaling=min-zoom&page-id=1%3A468). Please ensure to improve upon the mentioned aspects.

**Your mobile design:**
![image](https://user-images.githubusercontent.com/12745474/126993251-930cf013-3013-4d38-b2e3-0440abcf47e2.png)

**The original mobile design:**
![image](https://user-images.githubusercontent.com/12745474/126993808-22d9fed7-82e0-48bb-96ce-415f57da9637.png)